### PR TITLE
EL-3314 - List 3rd party dependencies for manual install

### PR DESCRIPTION
**Pull request by Sdhilipkumar**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3341

#### Description of Proposed Changes

Added installation instruction for 3rd party dependencies in the documentation sections.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-3341-list-3rd-party-components

#### Downstream Build(s)

* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-3341-list-3rd-party-components~CI/)

### CODE REVIEW

File: docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html [hunk 1]
```diff
@@ -63,9 +63,13 @@
...
 
-<p>
-    Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
-</p>
+<blockquote>
+    <p><strong>Note</strong>: If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually.</p>
+    <p>
+        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank">ng2-file-upload</a> for complete documentation on this component.
+    </p>
+</blockquote>
```
Comment (DeanHaire) [inline docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html:71]: I'm wondering would it be better to have these notes at the top of each of their sections so it's the first thing (except the actual example componenet) that a reader would see? Although for the file upload that would mean having two notes side by side which would look a bit weird so I'm not sure how you'd get around that. Maybe combine the notes into one? What do you think?

File: docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html [hunk 2]
```diff
@@ -63,9 +63,13 @@
     files into an area on the page.
 </p>
 
-<p>
-    Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
-</p>
+<blockquote>
+    <p><strong>Note</strong>: If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually.</p>
+    <p>
+        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank">ng2-file-upload</a> for complete documentation on this component.
```
Comment (DeanHaire) [inline docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html:69]:
```suggestion
        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank" rel="noopener">ng2-file-upload</a> for complete documentation on this component.
```
https://web.dev/external-anchors-use-rel-noopener/

**Review by DeanHaire — APPROVED** [review]

File: docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html [hunk 3]
```diff
@@ -52,8 +52,12 @@
 <hr>
 
 <blockquote>
+    <p><b>Note:</b> If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html:55]:
```suggestion
    <p><b>Note:</b> If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually.
```

File: docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html [hunk 4]
```diff
@@ -52,8 +52,12 @@
...
+        ( Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.)
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html:56]:
```suggestion
        Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
```

The two notes you have added should be consistent with each other.

File: src/package.json [hunk 5]
```diff
@@ -60,8 +60,6 @@
     "chart.js": "^2.9.3",
     "d3": "^5.9.2",
     "dragula": "^3.7.2",
-    "ng2-file-upload": "^1.3.0",
-    "ngx-mask": "^2.5.3",
```
Comment (donna-marijne) [inline src/package.json:64]: Because this is a breaking change, you should update BREAKING_CHANGES.md in both repositories with details on how to upgrade.

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Please create a downstream build for this branch.

**Comment by Sdhilipkumar** [comment]
As there is a conflict with the master branch hence I am closing this PR.

**Pull request closed without merging**